apps/backend/src/modules/ai-assistant/libs/helpers/suggest-task-by-category/suggest-task-prompt-messages.enum.ts

```diff
@@ -0,0 +1,39 @@
+const SuggestTaskPromptTemplates = {
+	SUGGEST_TASKS_CONTEXT: `
+The user has selected the following categories that are important for their personal growth and improvement.
+Each category represents an area where the user would like to make progress.
+Categories:
+`,
+	SUGGEST_TASKS_INSTRUCTIONS: `
+task:
+You should analyze the selected categories in combination with the user's responses from the onboarding process,
+and suggest specific and actionable tasks for each category that the user has selected for improvement.
+Ensure the tasks are realistic, motivational, and can be easily integrated into the user's daily routine.
+Provide no more than one task per category.
+
+response_structure:
+{
+	message: string,        // A message summarizing the suggested tasks.
+	tasks: [                // An array of tasks associated with each category.
+		{
+			categoryId: number,   // Unique identifier for the category.
+			categoryName: string, // Name of the category.
+			description: string,  // Detailed description of the task.
+			dueDate: string,      // Suggested due date for the task.
```

Comment:
```suggestion
```
AI shouldn't give us dueDate. We calculate it based on which days user has selected for tasks